test/extensions/filters/http/cache/cache_headers_utils_test.cc

```diff
@@ -0,0 +1,263 @@
+#include <chrono>
+#include <string>
+#include <vector>
+
+#include "envoy/common/time.h"
+
+#include "common/http/header_map_impl.h"
+
+#include "extensions/filters/http/cache/cache_headers_utils.h"
+
+#include "test/test_common/utility.h"
+
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Cache {
+namespace {
+
+#define DURATION(s) std::chrono::seconds(s)
```

Comment:
Please delete these macros. In this codebase, it's preferred to spell things out in every use.

Reply:
I only use them in this test file and I think without them the test cases will look way bigger, uglier, and less clear. Don't you agree?

Reply:
It doesn't matter whether I agree. These macros won't get accepted by the Envoy maintainers.

Reply:
I can't see anything in the style guide against it, I'd rather take a shot at it with the maintainers.

Reply:
@jmarantz, your call.

Reply:
Envoy style allows macros but these looks like they could be inline functions or constants. If you think they add a ton of value I'd recommend using those in general. For these cases:

IMO `DURATION(s)` obscures what's happening and what it replaces is not much longer.

For `UNSET_DURATION` you should just use `absl::nullopt`

`MAX_DURATION` .... again I think what it replaces is not too long and is clearer.

So I'm leaning against them, not because they are against style guide but because I think the code will be easier to read without the indirection.